README.md

```diff
@@ -20,7 +21,9 @@ This repository contains the source code for BinSkim, a Portable Executable (PE)
 3. Find out more about the Static Analysis Results Interchange Format **([SARIF](https://github.com/sarif-standard/sarif-spec/))** used to output Binskim results
 
 ### How to extract the exe file from the nuget package
```

Comment:
there two empty lines are done by the md file standard format, not manual change. I verified it looks the same from UI.